frontend/resources/styles/common/refactor/reorder.scss

```diff
@@ -0,0 +1,36 @@
+.reorder {
+  cursor: grab;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  left: var(--reorder-left-position, calc(-1 * $s-16));
```

Comment:
All stylesheets in `common/refactor` are deprecated, let's not add more CSS there.

We need to avoid using global classes for styles and use CSS modules. Since this CSS is meant to be shared among multiple components, it should be a mixin (see examples in `typography.scss`) or a placeholder selector (see `_buttons.scss`, for instance), and then import them from the relevant components.